diff --git a/Makefile b/Makefile
index 50f8408..4aeb98a 100755
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
-CC       = gcc
-LIBS     = -lreadline
-CFLAGS   = -O2 -g -Wno-unused-result -Wunused
+LIBS     = -static -static-libgcc -lreadline -ltinfo
+CFLAGS   = -O2 -g -Wno-unused-result -Wunused -Wl,-Bstatic
+
 
 OBJS     = hdlc.o  qcio.o memio.o chipconfig.o
 
@@ -24,47 +24,47 @@ ptable.o: ptable.c
 #	$(CC) -c qcio.c
 
 qcommand: qcommand.o  $(OBJS)
-	gcc $^ -o $@ $(LIBS)
+	$(CC) $^ -o $@ $(LIBS)
 
 qrmem: qrmem.o $(OBJS)
-	gcc $^ -o $@ $(LIBS)
+	$(CC) $^ -o $@ $(LIBS)
 
 qrflash: qrflash.o $(OBJS) ptable.o
-	gcc $^ -o $@ $(LIBS)
+	$(CC) $^ -o $@ $(LIBS)
 
 qwflash: qwflash.o $(OBJS)
-	gcc $^ -o $@ $(LIBS)
+	$(CC) $^ -o $@ $(LIBS)
 
 #qwimage: qwimage.o $(OBJS)
-#	gcc $^ -o $@ $(LIBS)
+#	$(CC) $^ -o $@ $(LIBS)
 
 qdload: qdload.o sahara.o $(OBJS)  ptable.o
-	gcc $^ -o $@ $(LIBS)
+	$(CC) $^ -o $@ $(LIBS)
 
 qwdirect: qwdirect.o $(OBJS)  ptable.o
-	gcc $^ -o $@ $(LIBS)
+	$(CC) $^ -o $@ $(LIBS)
 	
 qefs  : qefs.o efsio.o $(OBJS)
-	gcc $^ -o $@ $(LIBS)
+	$(CC) $^ -o $@ $(LIBS)
 
 qnvram  : qnvram.o $(OBJS)
-	gcc $^ -o $@ $(LIBS)
+	$(CC) $^ -o $@ $(LIBS)
 	
 mibibsplit: mibibsplit.o $(OBJS)
-	gcc $^ -o $@ $(LIBS)
+	$(CC) $^ -o $@ $(LIBS)
 
 qblinfo:    qblinfo.o $(OBJS)
-	gcc $^ -o $@  $(LIBS)
+	$(CC) $^ -o $@  $(LIBS)
 
 qident:      qident.o $(OBJS)
-	gcc $^ -o $@ $(LIBS)
+	$(CC) $^ -o $@ $(LIBS)
 
 qterminal:   qterminal.o $(OBJS)
-	gcc $^ -o $@ $(LIBS)
+	$(CC) $^ -o $@ $(LIBS)
 
 qbadblock:   qbadblock.o $(OBJS)  ptable.o
-	gcc $^ -o $@ $(LIBS)
+	$(CC) $^ -o $@ $(LIBS)
 
 qflashparm:  qflashparm.o $(OBJS)
-	gcc $^ -o $@ $(LIBS)
-	
\ No newline at end of file
+	$(CC) $^ -o $@ $(LIBS)
+	
